modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -95,87 +82,553 @@ The following charts are available only if you are using Unified Agent and have
+
+Web Services Failures::
+The average response time of Web Services connector requests, grouped by connector type in automatic intervals.
+
+FTP Requests::
+The average number of FTP connector requests, grouped by operation in automatic intervals.
+
+FTP Response Time::
+The average response time of FTP connector requests, grouped by connector type in automatic intervals.
+
+FTP Failures::
+The average number of FTP connector request failures, grouped by operation in automatic intervals.
+
+SFTP Requests::
+The average number of SFTP connector requests, grouped by operation in automatic intervals.
+
+SFTP Response Time::
+The average response time of SFTP connector requests, grouped by connector type in automatic intervals.
+
+SFTP Failures::
+The average number of SFTP connector request failures, grouped by operation in automatic intervals.
+
+SMTP Requests::
+The average number of SMTP connector requests, grouped by operation in automatic intervals.
+
+SMTP Response Time::
+The average response time of SMTP connector requests, grouped by connector type in automatic intervals.
+
+SMTP Failures::
+The average number of SMTP connector request failures, grouped by operation in automatic intervals.
+
+Object Store Requests::
+The average response time of Object Store connector requests, grouped by operation and automatic intervals.
+
+Object Store Response Time::
+The average number of Object Store connector request failures, grouped by operation and automatic intervals.
+
+Object Store Failures::
+The average number of Anypoint MQ connector inbound requests, grouped by operation and automatic intervals.
+
+Anypoint MQ Operations - Inbound::
+The average response time of Anypoint MQ connector inbound requests, grouped by operation and automatic intervals.
+
+Anypoint MQ Response Time - Inbound::
+The average number of Anypoint MQ connector inbound request failures, grouped by operation and automatic intervals.
+
+Anypoint MQ Failures - Inbound::
+The average number of Anypoint MQ connector outbound requests, grouped by operation and automatic intervals.
+
+Anypoint MQ Operations - Outbound::
+The average response time of Anypoint MQ connector outbound requests, grouped by operation and automatic intervals.
+
+Anypoint MQ Response Time - Outbound::
+The average number of Anypoint MQ connector outbound request failures, grouped by operation and automatic intervals.
+
+Anypoint MQ Failures - Outbound::
+The average number of requests of connectors of type other, grouped by connector type in automatic intervals.
+
+Other Connector Requests::
+The average number of requests of connectors of type other, grouped by connector type in automatic intervals.
+
+Other Connector Response Time::
+The average response time of requests for connectors of type other, grouped by connector type in automatic intervals.
+
+Other Connector Failures::
+The average number of requests failures of connectors of type other, grouped by connector type in automatic intervals.
```

Comment:
remove grouped by

@myuin-ms we should remove this chart.

Reply:
```suggestion
The average number of requests failures of connectors of type other, grouped in automatic intervals.
```